### DISCUSSION TITLE
Future Features

### DISCUSSION THREAD

**Discussion by briswells** [ChicoState/PantryNode#54] [Ideas]
I just thought it'd be a good idea to start writing down improvements to the applications as we're all working to retrofit it. I think having the ideas formulated will help future sprint planning.

 Personally, I would like to improve inventory tracking, which would allow users to log where an item is in the food bank.

**Comment by jayrevolinskyjr** [comment]
> I just thought it'd be a good idea to start writing down improvements to the applications as we're all working to retrofit it. I think having the ideas formulated will help future sprint planning.
> 
> Personally, I would like to improve inventory tracking, which would allow users to log where an item is in the food bank.

I think that's a great idea and could easily be integrated into the SKU lookup feature. What are your thoughts on specific improvements to the inventory tracking?

**Comment by ysiddhapura** [comment]
It would also be cool to categorize all the items, such as fruits, vegetables, snacks, etc.
